packages/server/src/api/controllers/view/utils.ts

```diff
@@ -53,12 +53,12 @@ export async function getViews() {
     }
   } else {
     const views = (
-      await db.allDocs(
+      await db.allDocs<any>(
```

Comment:
Again, wasn't sure what type this should be.

Reply:
Ah - this will return something like a `DesignDocument` - but it isn't well typed right now, same goes for the `db.get<any>` just above this - i'll jump on the branch and improve this.